Update diagram custom docs (#414)
DevExpress/devextreme-documentation: concepts/05 Widgets/Diagram/20 Custom Shapes/Custom Shapes.md

The **Diagram** widget provides a collection of built-in shapes. Use the [customShapes](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/customShapes '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/') option to extend this collection with custom shapes.

The [toolbox](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/toolbox '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/') option allows you to add the custom shapes to the toolbox.

> A custom shape can be created based on a default shape type or with a custom background image. Note, the [type](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/customShapes/#type '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#type') option identifies custom shapes, therefore it must be specified and unique.

Reviewer: ```suggestion
A custom shape can be created based on a default shape type or with a custom background image. The [type](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/customShapes/#type '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#type') option identifies custom shapes and should be unique.
```
